cEP-0010.md

```diff
@@ -0,0 +1,136 @@
+# coala Configuration
+
+| Metadata     |                                                |
+| ------------ |------------------------------------------------|
+| cEP          | 10                                             |
+| Version      | 1.0                                            |
+| Title        | Convert bears to use aspects                   |
+| Authors      | Asnel Christian Ngoulla <ngoullasob@gmail.com> |
+| Status       | Implementation Due                             |
+| Type         | Feature                                        |
+
+## Abstract
+
+This document describes how `aspects` will be implemented and bears' results
```

Comment:
Better *"... how `aspects` will be defined for and connected with bears and how their results will be ..."*